__tests__/components/editor/InputTarget.test.js

```diff
@@ -47,4 +47,12 @@ describe('<InputTarget />', () => {
   it('sets the typeahead component placeholder attribute', () => {
     expect(wrapper.find('#targetComponent').props().placeholder).toMatch('Frequency (RDA 2.14)')
   })
+
+  it('should call the onChange event and set the state with the selected option', () => {
+    const event = (wrap) => {
+      wrap.setState({options: ["{uri: 'URI', label: 'LABEL'}"]})
+    }
+    wrapper.find('#targetComponent').simulate('change', event(wrapper))
+    expect(wrapper.state().options[0]).toBe("{uri: 'URI', label: 'LABEL'}")
+  })
```

Comment:
ugh - I merged #229 first - does this need rebase so it's assoc with correct file?